project/src/main/java/frc/robot/commands/auto/DriveAtDistance.java

```diff
@@ -37,20 +50,38 @@ public void initialize()
   {
     endCommand = false;
     robot.drivetrain.stopMotors();
+    SmartDashboard.putString("Robot Mode:", "Drive At Distance");
+    DRIVE_SPEED = 0; 
   }
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute()
   {
-    robot.drivetrain.moveMotors(DRIVE_SPEED, 0);
+    DRIVE_SPEED = 0; 
+    distanceTimer.start(); 
+    double distance = Units.feetToMeters(4); 
```

Comment:
so this command will always drive 4 meters in the forward or backwards direction?